Shared/Extensions/JellyfinAPI/BaseItemDto+Images.swift

```diff
@@ -80,11 +80,16 @@ extension BaseItemDto {
         maxWidth: Int?,
         maxHeight: Int?,
         itemID: String
-    ) -> URL {
+    ) -> URL? {
+
         // TODO: See if the scaling is actually right so that it isn't so big
         let scaleWidth = maxWidth == nil ? nil : UIScreen.main.scale(maxWidth!)
         let scaleHeight = maxHeight == nil ? nil : UIScreen.main.scale(maxHeight!)
-        let tag = imageTags?[type.rawValue]
+        let tag = getImageTag(for: type)
+
+        if tag == nil && type != .backdrop {
```

Comment:
Can remove this and just have:
```swift
guard let tag else { return nil }
```

We don't need the check for backdrop images, if there aren't any then they won't populate the image tags. I only mentioned the "lack of index returning the primary image" as a lucky coincidence with the API.